### ISSUE TITLE
Show Spells and Items on Event Texts

### ISSUE THREAD

**Issue by Snowlite** [AscensionGameDev/Intersect-Engine#396]
**Is your feature request related to a problem? Please describe.**
When you make a quest and the NPC tells you the rewards you will receive. You can't see them. You can only read the name. 

**Describe the solution you'd like**
Be able to give the spells and items a code to use in the event texts so their picture shows up. 

An example would be:

White Potion: whitepot
Apple Juice: applejuice
Heal: healspell

Then the text event would be like:

"Thanks for helping me, I'm gonna give you this applejuice Apple Juice x1" showing you the respective item.

**Comment by Cheshire92** [comment]
I'm not sure that'd work exactly as you intend due to how these menus work.

What MIGHT be possible in the future however (though I make no promises here) is an option to add an item grid (roughly like the loot window in #385 ) as a second window showing configured items.

**Comment by Snowlite** [comment]
Yeah that works! even though something like this would be more close to what I meant:

![pic](https://user-images.githubusercontent.com/72949583/96356864-722a3280-10c2-11eb-9a1a-c244c07d2947.png) 

I don't know if that is possible? :o

**Comment by Cheshire92** [comment]
Anything is possible, whether it's what we'll do is a whole other story however. 😉
